packages/backend/src/open_payments/payment_pointer/service.ts

```diff
@@ -21,6 +23,9 @@ export interface PaymentPointerService {
   create(options: CreateOptions): Promise<PaymentPointer | PaymentPointerError>
   get(id: string): Promise<PaymentPointer | undefined>
   getByUrl(url: string): Promise<PaymentPointer | undefined>
+  addKeyToPaymentPointer(
```

Comment:
Can this instead be `paymentPointerKeysService.create`?
Either that or we could remove `PaymentPointerKeysService` and include the following here:
- `addKey`
- `getKeys`
- `revokeKey`